### PR TITLE
Loan - Debug & update

### PR DESCRIPTION
**Pull request by aspangaro**
2016-06-05 15_06_51-emprunt

### CODE REVIEW

File: htdocs/core/lib/loan.lib.php [hunk 1]
```diff
@@ -41,16 +41,6 @@ function loan_prepare_head($object)
 	$head[$tab][2] = 'card';
 	$tab++;
 
-    if (empty($conf->global->MAIN_DISABLE_NOTES_TAB))
```
Comment (eldy) [inline htdocs/core/lib/loan.lib.php:44]: Why did you remove the "note" tab ?

**Comment by aspangaro** [comment]
The "note" tab is already present in "card" tab, this page is now useless

**Comment by eldy** [comment]
We must follow the same rules than other objects: All notes of elements must be on a dedicated tab (with badge number to show if notes were filled or not). 
In a future, this tab may also contains more features like horodating and we will need a full tab for such things.

**Comment by hregis** [comment]
why not add the ability to have either all tabs in separate tabs or on a single page with ajax system show/hide?

**Comment by aspangaro** [comment]
@eldy I have restored the page. Thanks in advance to integrate.
